### DISCUSSION TITLE
Time zone of profiles

### DISCUSSION THREAD

**Discussion by dellafailleh** [NREL/ReEDS-2.0#75] [Q&A: Data] [answered]
Hello ReEDS Team,

First of all I admire the quality and precision of the ReEDS model! It is impressive and contains a lot of interesting data!
I would like to use the profiles for renewables and load defined here: "ReEDS_OpenAccess\inputs\variability"
Load and PV profiles are specified by balancing area while wind profiles are specified by resource region.

I have a question regarding the time zone convention. I see a mapping of balancing area to time zone defined here: 
"\ReEDS_OpenAccess\inputs\variability\reeds_ba_tz_map.csv"

Is this mapping also applicable for the wind profiles (using the 356 wind resource regions)?
What is the time zone convention for the different wind, pv and load profiles ? 

Many thanks!

Harold

**Comment by awfrazier** [comment, marked as answer]
Harold

These profiles are all in local time. When ReEDS is analyzing system operation at the hourly level, it is necessary to synchronize the data to a single unified time zone. The ba->tz map is used for that purpose.

The mapping is also applicable to wind. You'll find a wind resource region to ba map here: https://github.com/NREL/ReEDS_OpenAccess/blob/master/inputs/rsmap.csv

**Comment by dellafailleh** [comment]
Many thanks for your clear response @awfrazier !
